lib/cli/cordova.js

```diff
@@ -36,6 +36,13 @@ module.exports = function(argv, callback) {
     
     // stop cordova-cli from tracking us!
     cordovaCommand.push("--no-telemetry");
+    if(cordovaCommand[0] == "cordova" && 
+       cordovaCommand[1] == "plugin" && 
+       cordovaCommand.indexOf("--save") == -1 && cordovaCommand.indexOf("--no-save") == -1 &&
+       cordovaCommand[2] == "add"  || cordovaCommand[2] == "remove" || cordovaCommand[2] == "rm")
+    {
```

Comment:
When we auto save the platform we do a regex of the command to check what it is, I believe that will work better and help to simplify this conditional.

https://github.com/phonegap/phonegap-cli/blob/master/lib/phonegap/cordova.js#L207